.github/ISSUE_TEMPLATE/BUG-REPORT.yml

```diff
@@ -1,9 +1,6 @@
 name: Bug Report
 description: File a bug report
 title: "🐛 ISSUE TITLE"
-labels: ["bug", "triage"]
```

Comment:
do we need some default label, like "needs attention"?

Reply:
With the way we're currently tracking the metrics for this repo, it might complicate that. We intend to use the same format as is on the flutter repo (no labels by default) so it falls into our queue without any hitches